container-registry/task-execute-in-dind.yaml

```diff
@@ -17,6 +17,9 @@ spec:
     - name: resource-group
       description: target resource group (name or id) for the ibmcloud login operation
       default: ""
+    - name: registry-create-namespace
+      description: create container registry namespace if it doesn't already exists
+      default: true
```

Comment:
We put default value as plain string in the YAML as we hit some issues with tekton definitions in the past for boolean value
So I would expect `default: "true"`